Sources/Extensions/UIKit/UIViewController+Povio.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  UIViewController+Povio.swift
+//  PovioKit
+//
+//  Created by Toni Kocjan on 26/07/2020.
+//
+
+import UIKit
+
+public extension UIViewController {
+  class BarButton {
+    let content: Content
+    let action: Selector
```

Comment:
Should we add an optional `let target: Any?` here and then do `addTarget(barButton.target ?? self, ...`?
That way the caller can decide which object will receive the messages directly - edge case, so fine if we leave as-is 🤷